cmd/testing/base.go

```diff
@@ -29,18 +29,27 @@ import (
 
 type BaseSuite struct {
 	suite.Suite
-	Node     *node.Node
-	Client   *client.APIClient
-	ClientV2 clientv2.API
-	Host     string
-	Port     uint16
+	Node            *node.Node
+	Client          *client.APIClient
+	ClientV2        clientv2.API
+	Config          types.BacalhauConfig
+	Host            string
+	Port            uint16
+	AllowListedPath string
 }
 
 // before each test
 func (s *BaseSuite) SetupTest() {
 	logger.ConfigureTestLogging(s.T())
 
 	fsr, cfg := setup.SetupBacalhauRepoForTesting(s.T())
+	s.Config = cfg
+
+	// TODO: Update checker is configured with production default configs
+	//  and not respecting the test environment. This is a temporary fix
+	os.Setenv("BACALHAU_UPDATE_SKIPCHECKS", "true")
```

Comment:
So I hit this issue as well during some refactor work. I think the issue (and you may already be aware of this) is the config we provide to the devstack for BaseSuite is used by the _server_ but not the client - meaning the server isn't checking for updates, but the client is. This results in flaky tests since when output from a client command is inspected the update message is occasionally present.

I implemented a small mitigation for this via https://github.com/bacalhau-project/bacalhau/blob/7ecfa81ff4494b3f07e501111a56881950111ba7/cmd/testing/base.go#L86 which separates the stdout and stderr from the command result returned by the client. This allows tests to inspect stdout (where results are written) while ignoring stderr (where update messages are written)